### PR TITLE
[DOC] doxygen 1.9.5/1.9.6

### PR DESCRIPTION
**Pull request by eseiler**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -403,7 +403,7 @@ Header Changes:
   * If you used files as views with `seqan3::views::get<seqan3::field::id>` to project a single field, e.g.
     * `seqan3::views::get<seqan3::field::id>(fin)` => `std::views::transform(fin, [](auto && record){ return record.id(); })`
     * `fin | seqan3::views::get<seqan3::field::id>()` => `fin | std::views::transform([](auto && record){ return record.id(); })`
-    * or per projection: `fin | std::views::transform(&decltype(fin)::record_type::id)`
+    * or per projection: `fin | std::views::transform(&decltype(fin)::%record_type::id)`
```
Comment (smehringer) [inline CHANGELOG.md:406]: was this intentional?

Comment (eseiler) [reply CHANGELOG.md:406]: Escaping some doxygen auto-link generation: https://doxygen.nl/manual/autolink.html#linkfunc
(Which apparently didn't work before)

File: include/seqan3/io/sam_file/input_format_concept.hpp [hunk 2]
```diff
@@ -147,6 +147,12 @@ concept sam_file_input_format = requires (detail::sam_file_input_format_exposer<
                                 };
 //!\endcond
 
+#if SEQAN3_DOXYGEN_ONLY(1) 0
+template <typename t>
+class sam_file_input_format
+{};
+#endif
+
```
Comment (smehringer) [inline include/seqan3/io/sam_file/input_format_concept.hpp:155]: still missing a comment with the doxygen bug link

Comment (smehringer) [reply include/seqan3/io/sam_file/input_format_concept.hpp:155]: (all of the workarounds of that kind)

**Review by smehringer — CHANGES_REQUESTED** [review]

**Review by smehringer — APPROVED** [review]

**Review by SGSSGene — APPROVED** [review]

**Pull request merged**